api/routes/settings.py

```
@@ -48,12 +48,22 @@ def ensure_settings_dir():
             # Step 3: If not writable, TRY to fix it.
             try:
                 # 0o777 gives read/write/execute permissions to everyone.
+                # NOTE: on Windows os.chmod() is effectively a no-op for these
+                # Unix permission bits, so we verify it actually worked afterwards.
                 os.chmod(SETTINGS_DIR, 0o777)
```

Ruff (0.15.9)

[error] `os.chmod` setting a permissive mask `0o777` on file or directory

(S103)